frontend/model/state.js

```diff
@@ -216,6 +216,9 @@ const getters = {
   groupMembersCount (state, getters) {
     return getters.groupMembersByUsername.length
   },
+  groupShouldPropose (state, getters) {
+    return getters.groupMembersCount.length >= 3
```

Comment:
DRY: remember to update Invite.vue if you're going to add this

Reply:
done